mama/c_cpp/src/cpp/MamaSymbolList.cpp

```diff
@@ -177,6 +179,7 @@ namespace Wombat
         mamaSymbolList_setClosure             (myList, myPimpl);
         mamaSymbolList_setAddSymbolHandler    (myList, addMemberHandler);
         mamaSymbolList_setRemoveSymbolHandler (myList, removeMemberHandler);
+        // TODO: See why this is disabled
```

Comment:
Again, spin out an issue for investigation if we're not covering it in this release?